### DISCUSSION TITLE
12GB real files but 28GB on folders volumes... I don't understand, why ?

### DISCUSSION THREAD

**Discussion by Alexconquer** [seaweedfs/seaweedfs#4742] [Q&A]
Hello,

I don't understand files management in weed volume.

For me, i define the volumeSizeLimitMB 1024 and i see the trash represent 30%...
but my volume is x2 size.

Now, i have 33 volumes for 12GB files and my weed volume folders in 28GB.

i don't find the logic, help me ? ^^

**Comment by SmsS4** [comment]
check theses commands in `weed shell`
```shell
volume.vacuum
volume.fsck
volume.list
collection.list
fs.du # if you are using filer
```
probably 7 volumes are for logs if you are using `weed filer`

**Comment by Alexconquer** [comment]
thanks @SmsS4 :D 

I found : 

My folders size : 12GB
Topics folders : 37GB !!

I never imagined that metadata could take up so much space.

I use fuse mount via SeaweedFS for store my postgresql of Gitlab and my prometheus...

Is there a solution or am I using seaweedfs incorrectly?

**Reply by n9** [reply]
I am not sure whether is SeaweedFS the right storage for a relational database (changing data).

**Comment by Alexconquer** [comment]
Finally, i solve this with my other topic : https://github.com/seaweedfs/seaweedfs/discussions/4727#discussioncomment-7270439
